src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -284,6 +293,48 @@ private boolean verify(
         kubectl, manifestObjects, consoleLogger, verifyTimeoutInMinutes);
   }
 
+  /**
```

Comment:
Can you add a description for this method header?

Reply:
good catch